autopts/ptsprojects/mynewt/gatt.py

```diff
@@ -180,7 +180,14 @@ def set_addr(addr):
 
     custom_test_cases = [
         ZTestCase("GATT", "GATT/SR/GAN/BV-02-C",
-                  pre_conditions_2,
+                  pre_conditions_2 +
+                  [TestFunc(get_stack().synch.add_synch_element, [
+                      SynchPoint("GATT/SR/GAN/BV-02-C", 1),
+                      SynchPoint("GATT/SR/GAN/BV-02-C_LT2", 1)]),
+                  TestFunc(get_stack().synch.add_synch_element, [
+                      SynchPoint("GATT/SR/GAN/BV-02-C_LT2", 308),
+                      SynchPoint("GATT/SR/GAN/BV-02-C", 93)]),
```

Comment:
```suggestion
                  [TestFunc(get_stack().synch.add_synch_element, [
                            SynchPoint("GATT/SR/GAN/BV-02-C", 1),
                            SynchPoint("GATT/SR/GAN/BV-02-C_LT2", 1)]),
                  TestFunc(get_stack().synch.add_synch_element, [
                           SynchPoint("GATT/SR/GAN/BV-02-C_LT2", 308),
                           SynchPoint("GATT/SR/GAN/BV-02-C", 93)]),
```

Reply:
Trying to keep it unified with other places where SynchPoint is used like in here: (cap.py)
![obraz](https://github.com/user-attachments/assets/7edac34a-d37a-4547-83b2-27bfa67ace80)

Reply:
I think CAP is the outlier here; BAP, CSIP, Mesh, etc. align by the start parenthesis

Reply:
Ok :) In that case - padding is now changed.